repository/asset_psql.go

```diff
@@ -0,0 +1,73 @@
+package repository
+
+import (
+	"platform-go-challenge/models"
+)
+
+type AssetRepository struct{}
```

Comment:
What is the purpose of this empty struct type? Had you added some fields in this struct, what they would have been?

Reply:
My initial thought was to have a more flexible `AssetRepository` struct since the schema is not defined. More over, an empty struct has zero size and all pointers to it always point to the same special place in memory.